### PR TITLE
chore(Sentry): filter out Route Change

### PR DESCRIPTION
**Pull request by sethkfman**
## **Description**

This PR removes Route Change from being logged as a performance metric. Route Change is an event that does not provide any value when it comes to performance and is inflating the cost of performance monitoring. 

## **Related issues**

Fixes: https://github.com/MetaMask/mobile-planning/issues/1394

## **Manual testing steps**

1. Run application
2. See performance events logged and Route Change not being called

## **Screenshots/Recordings**

NA

### **Before**

NA

### **After**

[Log output video](https://recordit.co/BELvtdc4qF)

## **Pre-merge author checklist**

- [ ] I’ve followed [MetaMask Coding Standards](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/CODING_GUIDELINES.md).
- [ ] I've clearly explained what problem this PR is solving and how it is solved.
- [ ] I've linked related issues
- [ ] I've included manual testing steps
- [ ] I've included screenshots/recordings if applicable
- [ ] I’ve included tests if applicable
- [ ] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [ ] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.
- [ ] I’ve properly set the pull request status:
  - [ ] In case it's not yet "ready for review", I've set it to "draft".
  - [ ] In case it's "ready for review", I've changed it from "draft" to "non-draft".

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

**Review by tommasini — DISMISSED** [review]
LGtM

File: app/util/sentry/utils.js [hunk 1]
```diff
@@ -106,6 +106,22 @@ function rewriteReport(report) {
   return report;
 }
 
+/**
+ * This function excludes events from being logged in the performance portion of the app.
+ * @param {*} event - to be logged
+ * @returns event or null
```
Comment (digiwand) [inline app/util/sentry/utils.js:112]: [nit] 
```suggestion
 * @returns {(event|null)}
```

https://jsdoc.app/tags-returns
https://jsdoc.app/tags-param see: "Allows one type OR another type (type union)"

File: app/util/sentry/utils.js [hunk 2]
```diff
@@ -106,6 +106,22 @@ function rewriteReport(report) {
...
+ */
+function excludeEvents(event) {
+  //Modfiy or drop event here
```
Comment (digiwand) [inline app/util/sentry/utils.js:115]:
```suggestion
  //Modify or drop event here
```

**Review by digiwand — DISMISSED** [review]
nice update! minor comment suggestions

**Review by tommasini — APPROVED** [review]
LGTM

**Pull request merged**